lib/sycamore/sycamore/utils/nested.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any
```

Comment:
Seems odd that this would need to be its own file.

Reply:
I agree, but utils seems like the right place and none of the files in there looked like a good choice.